extensions/amp-story/1.0/amp-story.css

```diff
@@ -232,6 +252,43 @@ amp-story-page[active] {
   }
 }
 
+@keyframes i-amphtml-bookend-overlay-opacity-hidden {
+  0% {
+    opacity: 0.55;
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+.i-amphtml-story-opacity-mask {
+  position: absolute !important;
+  top: 0 !important;
+  bottom: 0 !important;
+  width: 100% !important;
+  height: 100% !important;
+  opacity: 0.55 !important;
+  /* Prevent someone from making this a full-screen image */
+  background-image: none !important;
+  background-color: #000 !important;
+  z-index: 100001 !important;
+  animation: i-amphtml-bookend-overlay-opacity 0.2s cubic-bezier(0.4, 0.0, 1, 1) forwards !important;
```

Comment:
You could actually do
`transition: opacity .... !important` (and remove the `forwards`), that'd allow you to remove the whole `@keyframes i-amphtml-bookend-overlay-opacity-hidden` block.

Also, it looks like you added the closing animation here, when this is actually the visible class. Could you please switch both animations duration/curves?